src/sql/parts/notebook/media/dark/cell_output_inverse.svg

```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"><defs><style>.cls-1{fill:#fff;}</style></defs><title>cell_output_inverse</title><path class="cls-1" d="M10.91,9.46v4H0V2.52H10.91v4h-1v-3H1v8.93H9.92v-3Zm3-2L11.8,5.35l.7-.7L15.83,8,12.5,11.3l-.7-.7,2.13-2.13H5v-1Z"/></svg>
```

Comment:
This file doesn't seem to be referenced. Are you missing a definition in the CSS file?

Reply:
Verified and deleted. Thought we would use them in future, but I am wrong, these icons are not even used in Aris extension.